### PR TITLE
feat(Popover): support VirtualElement as anchor

### PR DESCRIPTION
**Pull request by makhnatkin**

### CODE REVIEW

File: src/components/Popover/types.ts [hunk 1]
```diff
@@ -17,9 +18,11 @@ export type PopoverButtonProps = {
     onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
 };
 
+export type VirtualElement = popper.VirtualElement;
+
 export type PopoverExternalProps = {
     /** Allows to use custom anchor and disables `openByHover` and `onClick` for the component */
-    anchorRef?: React.RefObject<HTMLElement>;
+    anchorRef?: React.RefObject<HTMLElement | VirtualElement>;
```
Comment (amje) [inline src/components/Popover/types.ts:25]: Let's refactor to `PopupProps['anchorRef']`

File: src/components/Popover/types.ts [hunk 2]
```diff
@@ -17,9 +18,11 @@ export type PopoverButtonProps = {
     onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
 };
 
+export type VirtualElement = popper.VirtualElement;
```
Comment (amje) [inline src/components/Popover/types.ts:21]: There is no need to export this

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Popover/types.ts (hunk 1 above)
Comment (makhnatkin) [reply src/components/Popover/types.ts:25]: ok

File: src/components/Popover/types.ts (hunk 2 above)
Comment (makhnatkin) [reply src/components/Popover/types.ts:21]: ok

**Pull request closed without merging**